conda/pyhit/meta.yaml

```diff
@@ -1,5 +1,4 @@
-{% set build = 0 %}
-{% set version = "2023.10.24" %}
+{% set version = "2023.11.06" %}
```

Comment:
`conda/pyhit/conda_build_config.yaml` still needs to be updated as well to `moose-wasp 2023.11.06`.

Reply:
👍  good catch!